src/main/treeDataProviders/issuesTree/issuesTreeDataProvider.ts

```diff
@@ -555,7 +555,13 @@ export class IssuesTreeDataProvider implements vscode.TreeDataProvider<IssuesRoo
     ): Promise<void> {
         let cveToScan: string[] = [];
         descriptorNode.issues.forEach(issue => {
-            if (issue instanceof CveTreeNode && issue.cve && issue.cve.cve && !cveToScan.find(i => issue.cve && i == issue.cve.cve)) {
+            if (
+                issue instanceof CveTreeNode &&
+                !issue.parent.indirect &&
+                issue.cve &&
+                issue.cve.cve &&
+                !cveToScan.find(i => issue.cve && i == issue.cve.cve)
```

Comment:
I think this would work:
```suggestion
                !cvesToScan.includes(issue.cve?.cve)
```